### ISSUE TITLE
Add Breaker Panels

### ISSUE THREAD

**Issue by ellingtonisland** [Electrical-Age/ElectricalAge#909]
Add breaker Panels to electrical age that can control many circuits with breakers, like your main electrical panel does in your home.

this could be used to put machines and lights on individual circuits so you wouldn't have to cut power to a entire house.

**Comment by plajdo** [comment]
Isn’t it basically a wireless switch + relay connected together in a smart way?

**Comment by ellingtonisland** [comment]
except if you grounded or short circuited a circuit connected to it would pop the breaker in the panel instead of your main breaker or blowing up your wire.

**Comment by plajdo** [comment]
I’m not this mods’ developer or anything, but **maybe** I could try playing with this idea in my free time and make a pull request. But don’t expect anything as currently I have no idea how this mod works internally (&no time).

**Comment by jrddunbr** [comment]
The lamp supplies do this well enough already (including individual circuits), so adding a new block wouldn't make sense.

What would make sense just to add item slots for a breaker item in the lamp supplies, and maybe rename it to `Electrical Panel` or something like that.

**Comment by OmegaHaxors** [comment]
Yeah, lets go ahead and buff the Electrical Panel and make it the official power provider.

We'd have to balance it so that it isn't objectively better than running cable, but there's some design space there that needs filling. Maybe also add in a cable bus similar to the signal bus, only more limited. You'd only be able to attach up to four cables, they would interact with each other capacitivly, and share thermal code. It wouldn't be good for increasing power throughput (since sharing thermal power would reduce throughput across all of them) but would still be good for consolidating power across a single wire.

Use case: Put 4 cables together for a breaker box, then lay each one down towards your 4 machines. Now you can control each machine from a central location, but the amount of amperage along the line remains the same.
